README.md

```diff
@@ -28,7 +28,6 @@ Besides these, two DB backends are available:
 * Backtest
 * Candle
 * Credential
-* Market
```

Comment:
one remain in line 80.. it isn't used anyway

Reply:
good catch! fixed!